packages/ms-vc-api-issuer/README.md

```diff
@@ -0,0 +1,29 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Microsoft Azure Active Directory VC API Issuer (Typescript) 
```

Comment:
Please remove "API" from the name everywhere as it is too confusing with VC API, which is an actual W3C specification for a particaly API

Reply:
Ping

Reply:
changed